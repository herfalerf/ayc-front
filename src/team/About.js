import React from "react";
import ContactElement from "../contact/ContactElement";
import TeamList from "./TeamList";
import { makeStyles } from "@material-ui/core/styles";
import { Typography, Grid, Icon } from "@material-ui/core";
import ic_logomark from "../common/images/ic_logomark.svg";
import ic_about_curiosity from "../common/images/ic_about_curiosity.svg";
import ic_about_positivity from "../common/images/ic_about_positivity.svg";
import ic_about_sharing from "../common/images/ic_about_sharing.svg";
import ic_about_simplicity from "../common/images/ic_about_simplicity.svg";
import ScrollToTopOnMount from "../common/ScrollToTopOnMount";

// Show page with list of team members
//
// On Mount, loads team members from API.
//
// This is routed at /team
//
// Routes -> { TeamCard }
//

const useStyles = makeStyles((theme) => ({
  root: { flexGrow: 1 },
  upper: {
    backgroundImage: `url('${process.env.PUBLIC_URL}/images/about/about01.png')`,
    backgroundSize: "cover",
    backgroundPosition: "center bottom",
    paddingTop: "27%",
    width: "100%",
    height: "100%",
  },
  logo: {
    maxHeight: 40,
    margin: theme.spacing(2),
  },
  mainIcon: {
    height: 100,
  },
  aboutBox: {
    padding: theme.spacing(4),
  },
  aboutItem: {
    padding: theme.spacing(2),
  },

  teamBox: {
    backgroundColor: "rgb(244,245,248)",
    padding: theme.spacing(4),
  },
  teamMember: {
    padding: theme.spacing(2),
  },
  logoMark: { height: "14px", margin: "auto" },
  logoText: {
    fontWeight: "bold",
    fontSize: "20px",
    color: "#607494",
    margin: "2px",
  },
}));

function About() {
  console.debug("TeamList");

  const classes = useStyles();

  return (
    <div>
      <ScrollToTopOnMount />
      <div className={classes.upper}></div>
      <Grid container justifyContent="center" className={classes.aboutBox}>
        <Grid item xs={12}>
          <Typography align="center">
            <img
              src={process.env.PUBLIC_URL + "/images/logos/Logo03a.png"}
              alt="AYC Logo"
              className={classes.logo}
            ></img>
          </Typography>
        </Grid>
        <Grid item xs={12}>
          <Typography variant="h4" align="center">
            Forging Our Own Path
          </Typography>
        </Grid>
        <Grid item xs={12} sm={8} md={6}>
          <Typography variant="subtitle1" align="center">
            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Lorem
            ipsum dolor sit amet, consectetur adipisicing elit. Libero at
            voluptate officiis sed atque, ipsam quod est dicta veritatis eaque
            aut, sequi sapiente veniam eveniet aliquid totam? Labore, hic!
            Laboriosam!
          </Typography>
        </Grid>
      </Grid>
      <div className={classes.aboutBox}>
        <Grid container justifyContent="space-around" alignItems="center">
          <Grid container item lg={3} sm={6} className={classes.aboutItem}>
            <Grid item xs={12}>
              <Typography align="center">
                <Icon>
                  <img
                    src={ic_about_sharing}
                    alt="sharing icon"
                    className={classes.mainIcon}
                  />
                </Icon>
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="h5" align="center">
                Sharing knowledge
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="subtitle1" align="center">
                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
                Quidem, perspiciatis.
              </Typography>
            </Grid>
          </Grid>
          <Grid container item lg={3} sm={6} className={classes.aboutItem}>
            <Grid item xs={12}>
              <Typography align="center">
                <Icon>
                  <img
                    src={ic_about_curiosity}
                    alt="curiosity icon"
                    className={classes.mainIcon}
                  />
                </Icon>
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="h5" align="center">
                Curiosity
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="subtitle1" align="center">
                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
                Quidem, perspiciatis.
              </Typography>
            </Grid>
          </Grid>
          <Grid container item lg={3} sm={6} className={classes.aboutItem}>
            <Grid item xs={12}>
              <Typography align="center">
                <Icon>
                  <img
                    src={ic_about_simplicity}
                    alt="simplicity icon"
                    className={classes.mainIcon}
                  />
                </Icon>
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="h5" align="center">
                Simplicity
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="subtitle1" align="center">
                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
                Quidem, perspiciatis.
              </Typography>
            </Grid>
          </Grid>
          <Grid container item lg={3} sm={6} className={classes.aboutItem}>
            <Grid item xs={12}>
              <Typography align="center">
                <Icon>
                  <img
                    src={ic_about_positivity}
                    alt="positivity icon"
                    className={classes.mainIcon}
                  />
                </Icon>
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="h5" align="center">
                Positivity
              </Typography>
            </Grid>
            <Grid item xs={12}>
              <Typography variant="subtitle1" align="center">
                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
                Quidem, perspiciatis.
              </Typography>
            </Grid>
          </Grid>
        </Grid>
      </div>

      <div>
        <Grid container className={classes.teamBox}>
          <Grid container item xs={12} justifyContent="center">
            <Typography>
              <Icon className={classes.logoMark}>
                <img
                  src={ic_logomark}
                  alt="logo mark"
                  className={classes.logoMark}
                />
              </Icon>
              <Typography
                variant="subtitle1"
                className={classes.logoText}
                component="span"
              >
                WHO WE ARE
              </Typography>
            </Typography>
          </Grid>
          <Grid item xs={12}>
            <Typography variant="subtitle1" align="center">
              The Team Behind AlignYourCulture
            </Typography>
          </Grid>
          <TeamList />
        </Grid>
      </div>
      <ContactElement />
    </div>
  );
}

export default About;
