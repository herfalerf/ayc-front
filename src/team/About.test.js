import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import About from "./About";

it("renders without crashing", function () {
  render(
    <MemoryRouter>
      <About />
    </MemoryRouter>
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <About />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
