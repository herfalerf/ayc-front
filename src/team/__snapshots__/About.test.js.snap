// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="makeStyles-upper-20"
    />
    <div
      class="MuiGrid-root makeStyles-aboutBox-23 MuiGrid-container MuiGrid-justify-content-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          <img
            alt="AYC Logo"
            class="makeStyles-logo-21"
            src="/images/logos/Logo03a.png"
          />
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
        >
          Forging Our Own Path
        </h4>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6"
      >
        <h6
          class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Libero at voluptate officiis sed atque, ipsam quod est dicta veritatis eaque aut, sequi sapiente veniam eveniet aliquid totam? Labore, hic! Laboriosam!
        </h6>
      </div>
    </div>
    <div
      class="makeStyles-aboutBox-23"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-space-around"
      >
        <div
          class="MuiGrid-root makeStyles-aboutItem-24 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              <span
                aria-hidden="true"
                class="material-icons MuiIcon-root"
              >
                <img
                  alt="sharing icon"
                  class="makeStyles-mainIcon-22"
                  src="ic_about_sharing.svg"
                />
              </span>
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter"
            >
              Sharing knowledge
            </h5>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem, perspiciatis.
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-aboutItem-24 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              <span
                aria-hidden="true"
                class="material-icons MuiIcon-root"
              >
                <img
                  alt="curiosity icon"
                  class="makeStyles-mainIcon-22"
                  src="ic_about_curiosity.svg"
                />
              </span>
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter"
            >
              Curiosity
            </h5>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem, perspiciatis.
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-aboutItem-24 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              <span
                aria-hidden="true"
                class="material-icons MuiIcon-root"
              >
                <img
                  alt="simplicity icon"
                  class="makeStyles-mainIcon-22"
                  src="ic_about_simplicity.svg"
                />
              </span>
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter"
            >
              Simplicity
            </h5>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem, perspiciatis.
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-aboutItem-24 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              <span
                aria-hidden="true"
                class="material-icons MuiIcon-root"
              >
                <img
                  alt="positivity icon"
                  class="makeStyles-mainIcon-22"
                  src="ic_about_positivity.svg"
                />
              </span>
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter"
            >
              Positivity
            </h5>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem, perspiciatis.
            </h6>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class="MuiGrid-root makeStyles-teamBox-25 MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-content-xs-center MuiGrid-grid-xs-12"
        >
          <p
            class="MuiTypography-root MuiTypography-body1"
          >
            <span
              aria-hidden="true"
              class="material-icons MuiIcon-root makeStyles-logoMark-27"
            >
              <img
                alt="logo mark"
                class="makeStyles-logoMark-27"
                src="ic_logomark.svg"
              />
            </span>
            <span
              class="MuiTypography-root makeStyles-logoText-28 MuiTypography-subtitle1"
            >
              WHO WE ARE
            </span>
          </p>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter"
          >
            The Team Behind AlignYourCulture
          </h6>
        </div>
        <div
          class="LoadingSpinner"
        >
          Loading ...
        </div>
      </div>
    </div>
    <div
      class="makeStyles-root-31"
    >
      <div
        class="MuiGrid-root makeStyles-contactItem-32 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
           
          <h6
            class="MuiTypography-root makeStyles-thickText-34 MuiTypography-subtitle1 MuiTypography-alignCenter"
          >
            OUR RESULTS SPEAK FOR THEMSELVES
          </h6>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
           
          <h4
            class="MuiTypography-root makeStyles-contactSubTitle-35 MuiTypography-h4 MuiTypography-alignCenter"
          >
            Get In Touch to See How We Can Help
          </h4>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <h6
            class="MuiTypography-root makeStyles-contactText-36 MuiTypography-subtitle1 MuiTypography-alignCenter"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor facilis id aspernatur sit non quod fugit eaque similique consequatur tempora!
          </h6>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
           
          <p
            class="MuiTypography-root makeStyles-contactText-36 MuiTypography-body1 MuiTypography-alignCenter"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-33 MuiButton-containedPrimary"
              href="/contact"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiButton-label"
              >
                Send a Message
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
